contracts/SinglePlayerCommit.sol

```diff
@@ -51,213 +61,470 @@ contract SinglePlayerCommit is Ownable {
     event CommitmentEnded(address committer, bool met, uint256 amountPenalized);
     event Deposit(address committer, uint256 amount);
     event Withdrawal(address committer, uint256 amount);
```

Comment:
efficiency police: do we need a Withdrawal event? Could we get the same information from the event emitted by the token transfer?

Reply:
There are more things to review on the Events, like adding error events. To keep withing scope, do we want to revise them now?

Reply:
Created an Issue #16